src/legacy/layouts/Layout.js

```diff
@@ -2,6 +2,7 @@ import { StaticQuery, graphql } from "gatsby";
 import PropTypes from "prop-types";
 import React from "react";
 import { Helmet } from "react-helmet";
+import { TitleDescriptionPropType } from "../../PropTypeShapes";
```

Comment:
Let's avoid touching the stuff under `legacy`, since we'll be deleting it soon, once we do the real site launch.